securedrop/journalist_templates/_sources_confirmation_modal.html

```diff
@@ -0,0 +1,32 @@
+<div id="{{ modal_data.modal_id }}" class="menu-modal-dialog">
+  <a href="#close" class="external"></a>
+  <div id="delete-menu-dialog">
+    <div id="delete-menu-no-select">
+      <p class="modal-danger-text">
+      <span class="modal-danger-text">{{ gettext("Nothing Selected") }}</span>
+      </p>
+      <p>
+      {{ gettext("You must select one or more items for deletion.") }}
+      </p>
+    </div>
+    <div id="delete-menu-cta">
+    <p>
+      <span id="delete-menu-summary"></span>
+    </p>
+    <p>
+    {{ gettext("What would you like to delete?") }}
+    </p>
+    <p>
+      <button id="delete-files-and-messages" type="submit" name="action" value="delete-data" class="small btn danger">{{ gettext('Messages and Files') }}</button>
```

Comment:
You may have gone back and forth on this, but just noting that:
- The JI Content Audit records this message as "Files + Messages"
- Subsequent confirmation messages refer to "files and messages", so we're going from "Messages and Files" to "files and messages" at different steps of the process.

I'm not a big fan of the `+` in the original proposal but I would suggest making it consistent -- either  "messages and files" or "files and messages" throughout.

Reply:
Changing it back - again.